Update active/active documentation. (#2306)
hashicorp/terraform-website: content/enterprise/install/automated/active-active.mdx

- [Google Cloud - Network Quotas and Limits](https://cloud.google.com/vpc/docs/quota)
- [Google Cloud - API rate limits](https://cloud.google.com/compute/docs/api-rate-limits)

> Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for "sticky sessions." For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions be enabled to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Please reference the documentation for your infrastructure.

Reviewer: My only comment here is: Could we provide the briefest definition of "sticky session" for folks? Or are we assuming that everyone will be familiar with this already? Just checking!

Author: Okay, added another sentence w/a definition.

> Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for "sticky sessions." Sticky session is the common term for a setting in a load balancer or gateway device that ensures traffic is routed back to the original system that initiated a given request. For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions be enabled to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Please reference the documentation for your infrastructure.

Reviewer: ```suggestion
Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for sticky sessions. Sticky session refers to the practice of using a load balancer or gateway device to ensure traffic is routed back to the original system that initiated a request. For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Refer to the documentation for your infrastructure.
```
